paasta_tools/cli/cmds/spark_run.py

```diff
@@ -532,6 +534,21 @@ def get_spark_config(
     # Limit a container's cpu usage
     non_user_args['spark.mesos.executor.docker.parameters'] += ',cpus={}'.format(user_args['spark.executor.cores'])
 
+    if 'spark.eventLog.enabled' not in user_args:
+        user_args['spark.eventLog.enabled'] = 'true'
+
+    if 'spark.eventLog.dir' not in user_args and user_args['spark.eventLog.enabled'] == 'true':
```

Comment:
Line 494 "Update default configuration" overrides default user spark options with command line args.  Please move spark.eventLog.dir to user_args with defaults based on args.cluster.

Reply:
Thanks for the suggestion, already changed to that one :)